src/StreamJsonRpc.Shared/JsonRpc.cs

```diff
@@ -663,13 +663,14 @@ protected virtual Task<ReturnType> InvokeCoreAsync<ReturnType>(int? id, string t
         }
 
         /// <summary>
-        /// Creates a dictionary which maps a class method name to its JsonRpcMethodAttribute.Name values.
+        /// Creates a dictionary which maps a request method name to its clr method name via JsonRpcMethodAttribute value.
         /// </summary>
         /// <param name="target"></param>
         /// <returns></returns>
-        private static Dictionary<string, List<string>> GetMethodToAttributeMap(object target)
+        private static Dictionary<string, string> GetRequestMethodToClrMethodMap(object target)
         {
-            var methodToAttributeMap = new Dictionary<string, List<string>>(StringComparer.Ordinal);
+            // First we create a dictionary to map clr method names to JsonRpcmethodAttribute value, this simplifies the process of detecting attribute name conflicts.
+            var methodToAttributeMap = new Dictionary<string, string>(StringComparer.Ordinal);
```

Comment:
Consider updating the name of this variable to match what you're storing. Perhaps: `clrMethodToRequestMethodMap`